static/app/utils/replays/hooks/useDeleteReplay.tsx

```diff
@@ -19,6 +19,11 @@ export default function useDeleteReplay({projectSlug, replayId}: DeleteButtonPro
   const navigate = useNavigate();
   const organization = useOrganization();
 
+  const queryParams = new URLSearchParams(location.search);
+  const referrer = queryParams.get('referrer');
+  const groupId = queryParams.get('groupId');
+  const issueReferrer = !!referrer?.includes('issues');
```

Comment:
```suggestion
  const issueReferrer = Boolean(referrer?.includes('issues') && groupId)
```
To be extra sure groupId is non-empty below